adbc.h

```diff
@@ -1380,6 +1786,66 @@ AdbcStatusCode AdbcConnectionGetTableTypes(struct AdbcConnection* connection,
                                            struct ArrowArrayStream* out,
                                            struct AdbcError* error);
 
+/// \brief Get a view of types supported by the database.
+///
+/// The result is an Arrow schema with one field per type.  This represents
+/// the driver's mapping of the database type to an Arrow type.
```

Comment:
Should we clarify that the name of each field should be the database specific name of the type while the field's Type would be the mapped Arrow type?

Reply:
I don't want us to specify the casing, that's up to the database.

I'll clarify what the field name was meant to be.